docs/proxy_configuration.md

```diff
@@ -30,6 +30,50 @@ python neurons/validator.py \
     --wallet.hotkey your_hotkey
 ```
 
+## Brightdata Configuration
+
+BrightData provides a dedicated API for scraping Zillow property data with automatic proxy rotation and data normalization.
+
+### Setup Steps
```

Comment:
Can we define the exact steps to set up bright data, what tabs to click on etc. I know that it's not all that intuitive 

Data -> My scrapers -> New?

What did it take to set up a zillow data set?